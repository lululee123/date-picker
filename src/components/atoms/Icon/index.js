import React from 'react';
import classnames from 'classnames';

import styles from './index.scss';

const Icon = ({ className, onClick = () => {}, children }) => (
	<button type="button" onClick={onClick} className={classnames(styles.icon, className)}>
		{children}
	</button>
);

export default Icon;
