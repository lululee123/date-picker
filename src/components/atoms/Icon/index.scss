.icon {
    padding: 5px;
    width: 30px;
    height: 30px;
}