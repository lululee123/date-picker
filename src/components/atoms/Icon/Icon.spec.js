import React from 'react';
import renderer from 'react-test-renderer';

import Icon from './index';

test('render Icon correctly', () => {
	const component = renderer
		.create(
			<Icon />,
		)
		.toJSON();
	expect(component).toMatchSnapshot();
});
