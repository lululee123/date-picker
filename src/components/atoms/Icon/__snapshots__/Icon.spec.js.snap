// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Icon correctly 1`] = `
<button
  className="icon"
  onClick={[Function]}
  type="button"
/>
`;
